src/Models/Chat/Message.cs

```diff
@@ -106,6 +106,6 @@ public class Function
 		/// Gets or sets the arguments for the function, represented as a dictionary of argument names and values.
 		/// </summary>
 		[JsonPropertyName("arguments")]
-		public Dictionary<string, string>? Arguments { get; set; }
+		public IDictionary<string, object?>? Arguments { get; set; }
```

Comment:
I like to see this change, I was having a small problem for eventual non-string parameters returning from Ollama, happy to see this coming with the Extensions.AI support.